src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder employeeSalaryOfPeriod = new StringBuilder("Report for period "
+                + dateFrom + " - " + dateTo);
+        LocalDate dateBegin = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate dateEnd = LocalDate.parse(dateTo, FORMATTER);
+        dateBegin = dateBegin.minusDays(1);
+        dateEnd = dateEnd.plusDays(1);
+        for (String name : names) {
+            int totalSalary = 0;
+            for (int i = 0; i < data.length; i++) {
+                String[] temp = data[i].split(" ");
+                if (LocalDate.parse(temp[0], FORMATTER).isAfter(dateBegin)
```

Comment:
![Screenshot 2022-06-24 at 08 01 05](https://user-images.githubusercontent.com/60258373/175465656-517ac5ef-e25d-4568-963c-8989dbff49be.png)
Before you create PR with your homework, you need to go through the checklist under the task, and correct all the points described there. The mentor will not check the work until the checklist items are corrected